apps/beeai-ui/src/modules/agents/components/AgentDetailPanel.tsx

```diff
@@ -37,7 +34,8 @@ export function AgentDetailPanel() {
 
   if (!agent) return null;
 
-  const { name, description } = agent;
+  const { description, metadata } = agent;
+  const documentationUrl = metadata.links?.find(({ type }) => type === 'documentation')?.url;
```

Comment:
Helper:

```ts
function getLinkUrl<T extends components['schemas']['LinkType']>(
  links: components['schemas']['Metadata']['links'], type: T
): string | undefined {
  return links?.find((link) => link.type === type)?.url;
}
```

Use:

```ts
const documentationUrl = getLinkUrl(metadata.links, "documentation");
```

What do you think?

But it would be ideal, if Links would be a Record. Because then, we could do:

```ts
metadata.links?.Documentation?.url
```